### DISCUSSION TITLE
无法打开epub文件的Comic Book Images，求大神帮助~~！！

### DISCUSSION THREAD

**Discussion by PorkBoneSoup** [aooiuu/any-reader#54] [Q&A]
VSCode 插件 无法打开epub文件的Comic Book Images图片，不知道怎么解决，求大神帮助~~！！
![微信截图_20240815171522](https://github.com/user-attachments/assets/d9f9cff3-a707-4545-ba78-c73ff8129b21)

**Comment by aooiuu** [comment]
最好上传下 epub 文件于排查

**Reply by PorkBoneSoup** [reply]
大佬您好，这个就是我的出问题epub文件，请大佬排查。
[[Kox][望鄉太郎]話018-019.kepub.zip](https://github.com/user-attachments/files/16632146/Kox.018-019.kepub.zip)

**Reply by aooiuu** [reply]
v1.10.4 已修复

**Reply by PorkBoneSoup** [reply]
谢谢大佬~！感激不尽~！

**Reply by PorkBoneSoup** [reply]
大佬，我发现你开源的z-reader项目中也存在这个问题。

**Reply by aooiuu** [reply]
> 大佬，我发现你开源的z-reader项目中也存在这个问题。

z-reader 暂时没有精力维护
